<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>createcountdownwidget</class>
 <widget class="QDialog" name="createcountdownwidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>1080</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QComboBox" name="monthCombo">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>180</y>
      <width>72</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="dayCombo">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>180</y>
      <width>72</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="createButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>230</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Create Event</string>
    </property>
   </widget>
   <widget class="QComboBox" name="hourCombo">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>180</y>
      <width>72</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="yearCombo">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>180</y>
      <width>72</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="titleInput">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>130</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Enter Title Name</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
